robottelo/rhsso_utils.py

```diff
@@ -0,0 +1,121 @@
+"""Utility module to handle the rhsso-satellite configure UI/CLI/API testing"""
+import json
+import random
+
+from robottelo import ssh
+from robottelo.cli.base import CLIReturnCodeError
+from robottelo.config import settings
+from robottelo.constants import KEY_CLOAK_CLI
+from robottelo.constants import RHSSO_NEW_USER
+from robottelo.constants import RHSSO_RESET_PASSWORD
+from robottelo.datafactory import gen_string
+from robottelo.datafactory import valid_emails_list
+
+
+def run_command(cmd, hostname=settings.server.hostname, timeout=None):
+    """helper function for ssh command and avoiding the return code check in called function"""
+    if timeout is not None:
+        result = ssh.command(cmd=cmd, hostname=hostname, timeout=timeout)
+    else:
+        result = ssh.command(cmd=cmd, hostname=hostname)
+    if result.return_code != 0:
+        raise CLIReturnCodeError(
+            result.return_code, result.stderr, 'Failed to run the command : {}'.format(cmd),
+        )
+    else:
+        return result.stdout
+
+
+def get_rhsso_client_id():
+    """Getter method for fetching the client id and can be used other functions"""
+    client_name = "{}{}".format(settings.server.hostname, '-foreman-openidc')
+    run_command(
+        cmd="{0} config credentials "
```

Comment:
fstrings.

Reply:
@jyejare Avoiding fstrings here because I need ` settings.rhsso.host_url.replace("https://", "http://"),` replaced . It is becoming little ugly